include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -6120,6 +6120,12 @@
           "value" : "0x4000",
           "capabilities" : [ "VulkanMemoryModelKHR" ],
           "version" : "None"
+        },
+        {
+          "enumerant" : "Volatile",
```

Comment:
@dneto0 I missed this change yesterday. This should have a KHR suffix.

Reply:
Because Khronos already agreed to final semantics for this spelling, it is okay to not use the suffix; we get to skip the aliasing process in this situation.